browser/ui/views/brave_actions/brave_shields_action_view.cc

```diff
@@ -116,10 +119,18 @@ BraveShieldsActionView::GetImageSource() {
       badge = std::make_unique<IconWithBadgeImageSource::Badge>(
           badge_text, SK_ColorWHITE, kBadgeBg);
     }
+
+    SetTooltipText(l10n_util::GetStringUTF16(IDS_BRAVE_SHIELDS));
```

Comment:
BTW, how about handling tooltip by overriding `GetTooltipText(const gfx::Point& p)` instead of this method?

Reply:
yep, added

Reply:
Great! I don't have more :)